package.json

```diff
@@ -9,7 +9,7 @@
     "cypress:run": "cypress run",
     "cypress:open": "cypress open",
     "cypress:test": "start-server-and-test 'make integrationserver' http-get://localhost:8000 cypress:run",
-    "dev": "webpack --node-env development --watch",
+    "dev": "webpack --define-process-env-node-env development --watch",
```

Comment:
I'm just making sure this isn't a typo cause I haven't seen this before - the new option is called `--define-process-env-node-env`?

Reply:
@nikolas yea new webpack cli doesn't like the --node-env option. But I believe this does the same and works: [https://webpack.js.org/api/cli/#define-process-env-node-env](https://webpack.js.org/api/cli/#define-process-env-node-env)